dao/tx/tx.go

```diff
@@ -87,7 +95,11 @@ func NewTxModel(db *gorm.DB) TxModel {
 	}
 }
 
-func (*Tx) TableName() string {
+func (t *Tx) TableName() string {
+	if t.TxStatus <= StatusExecuted {
```

Comment:
I think this will not work as expected, check the form code:
![image](https://user-images.githubusercontent.com/7310198/189118717-c6ebab5b-3057-4943-b455-a26c1bd5213d.png)